Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.0.3'
+ruby '3.1.1'
```

Comment:
I think it's best to work with the same ruby version as the other collaborators of this project. The ruby version that was discussed to use was 3.0.3, so let's all kindly use it.